css-two-column-layout/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row text-align justify-center-content">
+      <div class="column-full">
+        <header>
+          <div class="column-full">
+            <h1>SCALE, SPEED, AGILITY</h1>
+            <p class="header-margin">We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently and
+            deliver an unparalleled quality of care.</p>
+          </div>
+        </header>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-half">
+        <div class="row">
+          <div class="column-half image-align">
+            <img alt="africa-outline" src="images/software.svg">
+          </div>
+          <div class="column-half">
+            <h3>INTUITIVE, SMART SOFTWARE</h3>
+            <p>We build products that transform how
+              healthcare facilities run across Africa
+              - making them radically more efficient
+              operationally and financially, a complete
+              turnaround.
+            </p>
+          </div>
+        </div>
+        <div class="row margin-cir">
+          <div class="column-half image-align">
+            <img alt="moving-lines" src="images/circumstance.svg">
+          </div>
+          <div class="column-half">
+            <h3>FOR ANY CIRCUMSTANCE</h3>
+            <p>Our intuitive system is built around the
+              peculiarities of your facility; we support
+              offline deployments, sync to the cloud
+              and run on any device 24/7.
+            </p>
+          </div>
+        </div>
```

Comment:
Why isn't this just another `column-half` like the one that precedes this? You should be able to have four `column-half` elements, one for each chunk in this row, and each of those chunks will have their own `row` inside of them
```
row
  column-half * 4
    row
      column-?
        img
      column-?
        text
```